tests/Cluster.py

```diff
@@ -129,7 +129,7 @@ def launch(self, pnodes=1, totalNodes=1, prodCount=1, topo="mesh", p2pPlugin="ne
         if self.staging:
             cmdArr.append("--nogen")
 
-        nodeosArgs="--max-transaction-time 50000 --abi-serializer-max-time-ms 50000 --filter-on * --p2p-max-nodes-per-host %d" % (totalNodes)
+        nodeosArgs="--max-transaction-time 50000 --abi-serializer-max-time-ms 65000 --filter-on * --p2p-max-nodes-per-host %d" % (totalNodes)
```

Comment:
I think we could have gone with even larger. Nothing in these tests require a small value.